docs/_sources/week02/project/index.rst.txt

```diff
@@ -4,112 +4,118 @@
 Week 2 - Project Week: Zika Mission Control
 ============================================
 
-Learning objectives for this week :ref:`week02-objectives`
-
-Project
-=======
-
-* Mission Briefing 1 : `pdf <../../_static/images/zika_mission_briefing_1.pdf>`_
+Zika Mission Control
+====================
 
+Throughout this class we will build on the Zika Mission Control project every single project week.
 
-Overview
-========
+This project will be our chance to practice the concepts we learned in the previous instruction week.
 
-Your goal is to build a web application to help CDC scientists visualize the reports of Zika infection in Central and South America. Using the concepts that you have learned from the past week, build a web application using Spring Boot that serves data to an OpenLayers map (starter code is provided below). Be sure that your web application is properly tested so that you can easily modify the application during the future weeks.
+Each week we will have a list of requirements that we will need to build into our project.
 
-Here are the user stories for your application.
+Project Overview
+================
 
-1. Scientists need to be able to populate the database with Zika reports that are in a CSV format.
-2. Scientists need to be able to see geographically where these infections are occurring so that they can respond with the appropriate assistance.
-3. Since each country reports data slightly differently, scientists need to be able to drill down into the data to understand what each report is saying.
+The Zika Mission Control dashboard will be a Spring web application. It will display an interactive map from OSM. The map will present a layer of features that represent an outbreak of Zika. Each representation of a Zika report will be clickable, and upon a click event the user will see more information about that specific report.
```

Comment:
Should OSM be defined? Maybe a link to the project and or link to OpenLayers doc describing it.

Reply:
Defined, and linked to openstreetmap.org.